build-tools/packages/build-cli/README.md

```diff
@@ -209,6 +210,36 @@ DESCRIPTION
 
 _See code: [@oclif/plugin-commands](https://github.com/oclif/plugin-commands/blob/v2.2.0/src/commands/commands.ts)_
 
+## `flub generate buildVersion [FILE]`
+
+This command is used to compute the version number of Fluid packages. The release version number is based on what's in the lerna.json/package.json. The CI will supply the build number and branch to determine the prerelease suffix if it is not a tagged build
+
+```
```

Comment:
Any language we should specify here?

Reply:
This is auto-generated by oclif.

Reply:
Okay. I thought maybe we might be able to configure it to spit out `bash` or something for better syntax highlighting. But it's not a big deal.